js/private/js_binary.bzl

```diff
@@ -213,7 +213,7 @@ def _create_launcher(ctx):
 
     output_data_files = copy_files_to_bin_actions(ctx, ctx.files.data, is_windows = is_windows)
 
-    bash_launcher = _bash_launcher(ctx, entry_point_path, ctx.attr.args)
```

Comment:
@gregmagolan something is wrong here, this last argument to `_bash_launcher` is never used, so usage of args is broken

Reply:
Ohh. I see. We are currently depending on bazel to pass arguments to the binary using the standard `args` method which works when running the binary `bazel run` and `bazel test`. In `nodejs_binary` we had a `templated_args` that allowed you to bake the args into the launcher script but we don't have that feature in `js_binary` currently. It seems like a smell to me since other rule sets don't allow you do bake arguments in.

Reply:
That call should just be
```
bash_launcher = _bash_launcher(ctx, entry_point_path)
```
and there doesn't have to be anything special done with `ctx.attr.args` for arguments to work with a binary or test rule

Reply:
Sorted it out. I added in fixed_args but only for rules that extend from js_binary_lib and not into the public API